inference-engine/tests/ngraph_helpers/ngraph_functions/src/utils/ngraph_helpers.cpp

```diff
@@ -388,6 +388,43 @@ std::vector<std::uint8_t> convertOutputPrecision(const std::vector<std::uint8_t>
                                                                                                         element::Type(toPrecision).get_type_name());
             }
         }
+        case element::Type_t::u32: {
```

Comment:
Are you sure, that these changes are required?

Reply:
No, they are not required. I will roll back to the original version.

Reply:
No, the changes are not required. I will roll back the ngraph_helpers.cpp to the original version.